products/deno.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Deno
+category: framework
+tags: javascript-runtime
+permalink: /deno
+versionCommand: deno --version
+releasePolicyLink: https://github.com/denoland/deno/releases
+changelogTemplate: https://github.com/denoland/deno/releases/tag/v__LATEST__
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:docker/denoland/deno
+-   purl: pkg:github/denoland/deno
+-   repology: deno
+
+auto:
+  methods:
+  -   git: https://github.com/denoland/deno.git
+
+releases:
+-   releaseCycle: "2"
+    releaseDate: 2024-10-09
+    eol: false
+    latest: "2.1.4"
+    latestReleaseDate: 2024-12-11
```

Comment:
Is v1 considered EOL? We should add v1 as well to the table.

Using `eoas` might also be helpful to split between Deno versions receiving active development or only bugfixes.

Reply:
Added v1 information. There's no guarantees that v1 is supported after it has become non-current (on the v2 release), therefore `eoas` not included. [Source](https://deno.com/blog/v1.46)